xml/System.Security.Cryptography/RSACng.xml

```diff
@@ -32,6 +32,9 @@
 -   The key used by <xref:System.Security.Cryptography.RSACng> is managed by a separate <xref:System.Security.Cryptography.CngKey> object. In contrast, <xref:System.Security.Cryptography.RSACryptoServiceProvider> has a key that is directly tied to the operations of the type itself.  
   
 -   <xref:System.Security.Cryptography.RSACng> performs such operations as signing and verifying signatures by using properties of the <xref:System.Security.Cryptography.RSACng> object, just as <xref:System.Security.Cryptography.ECDsaCng> uses its object properties to control signing and verification operations.  
+
+> [!NOTE]
+> The <xref:System.Security.Cryptography.RSACng> class is a CNG implementation for Windows platforms. In a cross-platform scenario, you should use the platform agnostic <xref:System.Security.Cryptography.RSA.Create%2A?displayProperty=fullName> to create an instance of the default implementation for the platform where your code will run.
```

Comment:
Something like  "The RSACng class is an implementation of the RSA algorithm using the Windows CNG libraries, and is not available on operating systems other than Windows. Applications which are not doing Windows interop are encouraged to use RSA.Create() instead of referencing this type directly."

"The RSAOpenSsl class is an implementation of the RSA algorithm using OpenSSL. It is not available on Windows, and is only available on other operating systems when OpenSSL is installed. Applications which are not doing OpenSSL-specific interop are encouraged to use RSA.Create() instead of referencing this type directly."